src/Microsoft.SqlTools.Hosting/Utility/CommandOptions.cs

```diff
@@ -144,5 +141,16 @@ private void SetLocale(string locale)
                 // into the CultureInfo constructor.
             }
         }
+
+        public void LocaleSetter(string locale)
```

Comment:
Is this for testing? Consider making internal. At a minimum I'd suggest changing to `SetLocalUnsafe` or something, this isn't a great name.

Reply:
I separated this out mainly for code re-use in ServiceLayerCommandOptions as well. It sets the global culture, which is used in string formatting for messages. 'Locale' itself is used in service layer tests, otherwise is unused. I'll add in a summary stating so and make it internal.